environment/src/test/java/com/sequenceiq/environment/environment/v1/EnvironmentViewConverterTest.java

```diff
@@ -15,21 +15,22 @@
 import static com.sequenceiq.environment.environment.service.EnvironmentTestData.newTestEnvironment;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
 
+import com.sequenceiq.environment.credential.v1.converter.CredentialViewConverter;
 import com.sequenceiq.environment.environment.EnvironmentStatus;
 import com.sequenceiq.environment.environment.domain.Environment;
 import com.sequenceiq.environment.environment.domain.EnvironmentView;
+import com.sequenceiq.environment.environment.domain.EnvironmentViewConverter;
 
+@ExtendWith(MockitoExtension.class)
 class EnvironmentViewConverterTest {
 
-    EnvironmentViewConverter underTest;
+    private final CredentialViewConverter credentialViewConverter = new CredentialViewConverter();
```

Comment:
should be mocked

Reply:
The test was originally testing if the crn of credential of the environment is converted. There are 2 choices:
1) I delete this assertion and mock the cred converter
2) I leave the assertion and the actual cred converter implementation in it's place

Which one should be?